Demo/Demo/Info.plist

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>UIAppFonts</key>
```

Comment:
Do we have a ticket to add documentation for this for merchants? It seems like this is something that a merchant would need to do to use our fonts. What happens if they don't add this to their `Info.plist`?

Reply:
This line was added in the CHANGELOG under Payment Buttons so that merchants know about the font change: `Font typeface changed to "PayPalOpen" to meet brand guidelines`

The font shouldn't be in `Info.plist` so I've removed it. The font is registered in the SDK under the `PaymentButtonFont` file so that merchants won't need to add the font themselves